src/extension/serializer.ts

```diff
@@ -991,18 +1035,33 @@ export class GrpcSerializer extends SerializerBase {
     token: CancellationToken,
   ): Promise<Serializer.Notebook> {
     const identity = this.lifecycleIdentity
-    const deserialRequest = DeserializeRequest.create({ source: content, options: { identity } })
-    const request = await this.client!.deserialize(deserialRequest)
+    const deserializeRequest = new DeserializeRequest({
+      source: content,
+      options: { identity },
+    })
+    let res
+    try {
+      res = await this.client.deserialize(deserializeRequest)
```

Comment:
I added extra error handling here because without it there was no error shown in vscode when the the request for deserialization failed. the old grpc serializer didn't do any additional error checking ... i can remove it if we feel safe to do so